README.md

```diff
@@ -44,278 +44,316 @@ Unrecommended ones can still be considered. The most important thing is to remai
+-  context • 🟢 ctx
+-  continue • 🟢 cont
+-  control • 🟢 ctrl
+-  conversation • 🟢 conv
+-  coordinate • 🟢 coord
+-  copy • 🔴 cpy
+-  cosine • 🟢 cos
+-  cotangent • 🟢 cot
+-  counter • 🟢 cnt
+-  current
+   -  🟢 cur
+   -  🟢 curr
 
 ### D
 
--  🟢 db • database
--  🟢 dbg • debug
--  🟢 dec • decimal / decrease
--  🟢 def • default / define
--  🟢 del • delete
--  🟢 dest • destination
--  🟢 dev • development / device / developer
--  🟡 diff • difference {operation}
--  🟢 dir • directory / direction
--  🟢 dis • disable
--  🟢 disp • display
--  🟢 doc • document
--  🟢 drv • driver
--  🟢 dsc • descriptor
--  🟡 dt • delta time {calculation}
--  🟢 dyn • dynamic
+-  databse • 🟢 db
+-  debug • 🟢 dbg
+-  decimal • 🟢 dec
+-  decrease • 🟢 dec
+-  default • 🟢 def
+-  define • 🟢 def
+-  delete • 🟢 del
+-  delta time • 🟡 dt {calculation}
+-  descriptor • 🟢 dsc
+-  destination • 🟢 dest
+-  developer • 🟢 dev
+-  development • 🟢 dev
+-  device • 🟢 dev
+-  difference • 🟡 diff {operation}
+-  direction • 🟢 dir
+-  directory • 🟢 dir
+-  disable • 🟢 dis
+-  display • 🟢 disp
+-  document • 🟢 doc
+-  driver • 🟢 drv
+-  dynamic • 🟢 dyn
 
 ### E
 
--  🟢 e • event
--  🟢 e.g. • example
--  🟢 en • enable
--  🟢 env • environment
--  🟡 eq • equal {binary operation}
--  🟢 err • error
--  🟢 exe • executable
--  🟢 expr • expression
--  🟢 ext • extension
--  🟢 etc • etcetera
+-  enable • 🟢 en
+-  environment • 🟢 env
+-  equal • 🟡 eq {binary operation}
+-  error • 🟢 err
+-  etc • 🟢 etc
+-  event • 🟢 e
+-  executable • 🟢 exe
+-  expression • 🟢 expr
+-  extension • 🟢 ext
 
 ### F
 
--  🟢 f / func • function
--  🟢 fac • factory
--  🟢 fd • file descriptor
--  🟢 fig • figure
--  🟢 fmt • format
--  🟢 fp • function pointer
+-  factory • 🟢 fac
+-  file descriptor • 🟢 fd
+-  figure • 🟢 fig
+-  for example 🟢 e.g.
+-  format • 🟢 fmt
+-  function
+   -  🟢 f
+   -  🟢 fn
+   -  🟢 func
+-  function pointer • 🟢 fp
 
 ### G
 
--  🟡 ge • greater or equal {binary operation}
--  🟢 gen • generate
--  🟡 gt • greater than {binary operation}
+-  generate • 🟢 gen
+-  greater or equal • 🟡 ge {binary operation}
+-  greater than • 🟡 gt {binary operation}
 
 ### H
 
--  🟡 h • height {shape}
--  🟢 hex • hexadecimal
--  🔴 hdr • header
--  🟡 hor • horizontal {shape}
--  🟡 hw • hardware {computer science}
+-  hardware • 🟡 hw {computer science}
+-  header • 🔴 hdr
+-  height • 🟡 h {shape}
+-  hexadecimal 🟢 hex • hexadecimal
+-  horizontal • 🟡 hor {shape}
 
 ### I
 
--  🟢 id • identifier
--  🟢 idx • index
--  🟢 iface • interface
--  🟢 img • image
--  🟢 in • input
--  🟢 inc • include / increase
--  🟢 info • information
--  🟢 init • initialize
--  🟢 ins • insert
--  🟢 intf • interface
--  🟢 int • integer
--  🟢 iter • iterator
+-  identifier • 🟢 id
+-  image • 🟢 img
+-  include • 🟢 inc
+-  increase • 🟢 inc
+-  index • 🟢 idx
+-  information • info
+-  initialize • 🟢 init
+-  initialization • 🟢 init
+-  input 🟢 in • input
+-  insert • 🟢 ins
+-  integer • 🟢 int
+-  iterator • 🟢 iter
+-  interface
+   -  🟢 iface
+   -  🟢 intf
```

Comment:
IDK which is better, needs some research.

Reply:
Idk too